dashboard/db/migrate/20160429214539_add_script_to_course_units.rb

```diff
@@ -0,0 +1,6 @@
+class AddScriptToCourseUnits < ActiveRecord::Migration
+  def change
+    Plc::CourseUnit.destroy_all
```

Comment:
Discussed offline — I will manually destroy all PLC objects from each environment, and remove this line from the migration.  **Confirmed with Brook that all PLC content in Production can be irreversibly destroyed.**